libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotFrameworkServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Bot.Builder.Integration.AspNet.Core.StreamingExtensions
+{
+    /// <summary>
+    /// Used to register the 'front door' adapter that selects which adapter is needed to process an incoming HTTP request.
+    /// GET requests asking for WebSocket upgrades are directed to a Bot Framework Protocol Version 3 with Streaming Extensions enabled adapter.
+    /// POST requests are directed to a traditional Bot Framework adapter.
+    /// </summary>
+    public static partial class BotFrameworkServiceCollectionExtensions
+    {
```

Comment:
Why is this a partial class? Seems like it should be a "normal" static class.

Reply:
After checking with the team we think this was left over from an earlier design and isn't needed in the current version. I'll remove the entire file. Hooray for less surface space!